app/jobs/attach_files_to_work_job.rb

```diff
@@ -14,6 +14,7 @@ def perform(work, uploaded_files)
       attach_content(actor, uploaded_file.file)
       uploaded_file.update(file_set_uri: file_set.uri)
     end
+    InheritPermissionsJob.perform_later(work)
```

Comment:
@awead I was reusing tested code, but I can see the point that it would be more efficient to just set it.  Since there was already an update to the test to check for permission changes I updated the code.